src/NSwag.AspNetCore.Launcher/Program.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace NSwag.AspNetCore.Launcher
 {
-    public class Program
+    internal class Program
```

Comment:
Since the launcher binary will now be added as a reference when a package reference to NSwag.ConsoleCore, I made this internal to avoid polluting the namespace. It's an internal artifact, so a user project isn't expected to call this.